napari/_qt/qt_main_window.py

```diff
@@ -428,10 +428,36 @@ def _add_file_menu(self):
         closeAction.setShortcut('Ctrl+W')
         closeAction.triggered.connect(self._qt_window.close)
 
+        from ..plugins import _sample_data
+
+        open_sample_menu = QMenu(trans._('Open Sample'), self._qt_window)
+        for plugin_name, samples in _sample_data.items():
+            multiprovider = len(samples) > 1
+            if multiprovider:
+                menu = QMenu(plugin_name, self._qt_window)
+                open_sample_menu.addMenu(menu)
+            else:
+                menu = open_sample_menu
+
+            for samp_name in samples:
+                key = (plugin_name, samp_name)
```

Comment:
yeah, I prefer the space in the menu too.  So, happy to make the change if @jni has no major objections

Reply:
This is awesome work! I would love to be able to do this, as I am not sure where to look for examples sometimes! 💯 

Now, question. How would this work with localization?

There should be an `ID` `viewer.open_sample('scikit-image', 'human mitosis')` in  this case `human mitosis` but we would also want a human readable display name in the localized language.

Reply:
Yeah, good point @goanpeca it would be nice to be able to translate these strings too

Reply:
> How would this work with localization?

ooof!  😖  very good question!  Since this is plugin-provided, how should we go about thinking about that?

Reply:
Well plugin devs should handled their own translations, that is they are in charge of doing the whole

`trans._()` on the things they provide. Now I do not know if the hook spec provides a way of having an ID and a "displayName", since this is going to become something we need to provide in several places where we need a unique identifier AND a display version of it 🤔

Reply:
Works for me :)

Reply:
updated with localization.  The skimage example shows a bit more detail in the menus now (but not size):

<img width="231" alt="Untitled" src="https://user-images.githubusercontent.com/1609449/113355989-9ba4e680-930f-11eb-86b4-eba50731b4d9.png">

Reply:
SO COOL.

happy with the human-readable and localizable names in general. =) I don't love the new nested dictionary but I don't hate it either, and I don't see a better way to do it. So 👍 all around!